scss/os/_overrides_bootstrap.scss

```diff
@@ -156,3 +156,10 @@ ul {
     margin-bottom: 0;
   }
 }
+
+//sass-lint:disable no-duplicate-properties, no-vendor-prefixes
```

Comment:
Please include a note on why this is needed, so it isn't removed if someone is working on #141. We prefer to avoid vendor prefixes, but this doesn't have an alternate solution.

Reply:
sass-lint:disable-block has been added